modules/travel_pay/app/controllers/travel_pay/application_controller.rb

```diff
@@ -26,7 +26,7 @@ class ApplicationController < ::ApplicationController
     # * Remove this before_action
     # * Remove block_if_flag_disabled definition
 
-    before_action :block_if_flag_disabled
+    before_action :feature_enabled
```

Comment:
renaming for consistency with other methods making this check in `vets-api`

Reply:
There's only 9 instances in the codebase where `feature_enabled` is used. I thought `block_if_flag_disabled` better explained the before action, but happy to chat about this.